src/constants/constants.ts

```diff
@@ -5,4 +5,5 @@ export const COMPLETED = 'COMPLETED';
 export const DONE = 'DONE';
 export const AVAILABLE = 'AVAILABLE';
 export const UNASSINGED = 'UNASSINGED';
+export const PROGRESS_SUCCESSFUL = 'Progress Updated Successfully';
```

Comment:
NIT: we can create this message inside an object, this will create a separate context for messages related to PROGRESS and we can add more messages inside this object.
Feel free to ignore this suggestion if this is not under the scope of change.

```suggestion
export const PROGRESS_MESSAGE = {
  UPDATE_SUCCESS = 'Progress Updated Successfully'
};
```